homeassistant/components/tuya/select.py

```diff
@@ -236,6 +236,16 @@
             icon="mdi:layers-outline",
         ),
     ),
+    # Thermostat
+    # https://developer.tuya.com/en/docs/iot/f?id=K9gf45ld5l0t9
+    "wk": (
+        SelectEntityDescription(
+            key=DPCode.VALVE_SET,
```

Comment:
I cannot find this data point in the linked API specification documentation above, could point me to the right location?

Reply:
As above - shared the API response. I cannot really be responsible for the accuracy of Tuya documentation.

Reply:
It seems like an issue, as in, the device is not communicating according to specifications. Have you contacted/reported it with Tuya?

Reply:
Please not comment on Standard Instruction set.

No reply from Tuya support

Reply:
@majduk Did they finally respond?